src/pcp/js/exemplars.js

```diff
@@ -0,0 +1,201 @@
+/*
+
+Copyright 2013-2017 OCAD University
+
+
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+
+BSD license. You may not use this file except in compliance with one these
+
+Licenses.
+
+
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+
+*/
+
+/* global fluid */
+
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.pcp");
+
+
+    /**
+     * Represents an "exemplar" (configuration) for a component.
+     * A good place to keep a *related template resource* path.
```

Comment:
Expand description a bit to list the other things held by an exemplar - schema type etc. and document the resulting options structure better.